src/arch/riscv/reg/riscv.rs

```diff
@@ -0,0 +1,70 @@
+//! RISC-V Register Definitions.
+//!
+//! Useful links:
+//!     - [GNU binutils-gdb XML Descriptions](https://github.com/bminor/binutils-gdb/blob/master/gdb/features/riscv)
```

Comment:
Could you move these links down into the doc comment for `RiscvCoreRegs`? 
I'd like to keep things consistent with the other implementations.

Also, since the module isn't marked `pub`, this doc-comment won't actually correspond to any generated docs, so it can just be removed.